### PR TITLE
[update] update README.md

### PR DESCRIPTION
**Pull request by csungwon**
Updated the design of main models. Added instruction on how to setup
environment variables (`github_key`, `github_secret`). Added a list
of main features from this engagement.

### CODE REVIEW

**Review by kimhoang73 — APPROVED** [review]
thanks for updating the readme! we definitely needed to add the steps to set up the environment variables to get the app running locally. also, good idea to add the main features we implemented

**Review by kyriakipapadimitriou — APPROVED** [review]
This looks great! Thanks for putting it together and illustrate the Oauth updates and new features! 
Very concise and clear descriptions

File: README.md [hunk 1]
```diff
@@ -10,8 +10,7 @@ Engineering](https://cs169.saas-class.org).
 [![Test Coverage](https://codeclimate.com/github/csungwon/esaas-engagements/badges/coverage.svg)](https://codeclimate.com/github/csungwon/esaas-engagements/coverage)
 
 [Pivotal Tracker](https://www.pivotaltracker.com/n/projects/2070245)
-
-Heroku Link: https://shielded-bastion-61752.herokuapp.com/
+[Heroku Deployment](https://shielded-bastion-61752.herokuapp.com/)
```
Comment (Jldeatrick) [inline README.md:13]: Maybe before demo day we should change the heroku URL like group 21 did with theirs. Not necessary by any means but looks nice

**Review by Jldeatrick — APPROVED** [review]
Very nice changes. The summary section on our additions looks great

File: README.md (hunk 1 above)
Comment (csungwon) [reply README.md:13]: Yeah I thought about that, but the golden repo already took it. (esaas-engagements.herokuapp.com). I can do something similar (e.g. essas-engagemtns-group20) if you guys want.

**Pull request merged**